string plate_number
float64 speed